articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+2. In the **New** blade, search for `Local network gateway`.
+
+   ![Create new networking resource](images/azs-public-browser-new-local-network-gateway-search.png)
+
+3. In the **Local network gateway** blade, click **Create**.
+
+    ![Create button in local network gateway blade](images/azs-public-browser-new-local-network-gateway-create.png)
+
+4. In the **Create local network gateway** blade, enter the following information:
+
+   - **Name** - The name of the local network gateway.
+
+   - **IP address** - This is the public IP address of the VPN device that you want Azure to connect to. Specify a dummy IP address, as this will need to updated once you have generated the public IP with the connection.
+
+   - **Address space** - One or more IP address ranges (in CIDR notation) that define your local network's address space. For example: 192.168.0.0/16. If you plan to use this local network gateway in a BGP-enabled connection, then the minimum prefix you need to declare is the host address of your BGP Peer IP address on your VPN device.
+
+   - **Configure BGP settings** - Use only when configuring BGP.
+
+   - **Subscription** - This is your Microsoft Azure subscription.
+
+   - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   - **Location** - Select the location of the Azure resource.
+
+     ![Create new local network gateway](images/azs-public-browser-create-local-network-gateway-blade.png)
+
+5. Click **Create**.
+
+## Create the VPN connection in Public Azure
+
+Create the Site-to-Site VPN connection between your virtual network gateway and your Azure Stack VPN:
+
+1. Navigate to your virtual network gateway by clicking **All services**, then select **Virtual network gateways** under the networking section.
+
+    ![Navigate to the Virtual network gateway](images/azs-public-browser-services-virtual-network-gateway.png)
+
+2. Select your virtual network gateway from the list.
+
+3. In the *new* blade for your virtual network gateway, select **Connections** under the **Settings** section.
+
+    ![Connections for virtual network gateway](images/azs-public-browser-virtual-network-gateway-connections-button.png)
+
+4. Click **Add**.
+
+5. In the **Add connection** blade, enter the following information:
+
+   - **Name** - The name of the connection.
+
+   - **Connection type** - Select **Site-to-site (IPSec)**.
+
+   - **Virtual network gateway** - Select the virtual network gateway you created earlier.
+
+   - **Local network gateway**- Select the local network gateway you created earlier.
+
+   - **Shared Key** - A mixture of letters and numbers, used to establish encryption for the connection. You must use the same shared key in both the virtual network and local network gateways. If your gateway device doesn't provide one, you can make one up here and provide it to your device.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
```

Comment:
In this instance (Public Azure), should this be:
This is your Microsoft Azure subscription.

Reply:
Changed to: This is your Public Azure subscription.